editor/script_create_dialog.cpp

```diff
@@ -269,40 +266,46 @@ void ScriptCreateDialog::_file_selected(const String &p_file) {
 
 void ScriptCreateDialog::_path_changed(const String &p_path) {
 
-	path_valid = false;
+	is_path_valid = false;
+	is_new_script_created = true;
 	String p = p_path;
 
 	if (p == "") {
-
-		path_error_label->set_text(TTR("Path is empty"));
-		path_error_label->add_color_override("font_color", Color(1, 0.4, 0.0, 0.8));
+		_msg_path_valid(false, TTR("Path is empty"));
+		_update_dialog();
 		return;
 	}
 
 	p = GlobalConfig::get_singleton()->localize_path(p);
 	if (!p.begins_with("res://")) {
-
-		path_error_label->set_text(TTR("Path is not local"));
-		path_error_label->add_color_override("font_color", Color(1, 0.4, 0.0, 0.8));
+		_msg_path_valid(false, TTR("Path is not local"));
+		_update_dialog();
 		return;
 	}
 
 	if (p.find("/") || p.find("\\")) {
 		DirAccess *d = DirAccess::create(DirAccess::ACCESS_RESOURCES);
-
 		if (d->change_dir(p.get_base_dir()) != OK) {
-
-			path_error_label->set_text(TTR("Invalid base path"));
-			path_error_label->add_color_override("font_color", Color(1, 0.4, 0.0, 0.8));
+			_msg_path_valid(false, TTR("Invalid base path"));
 			memdelete(d);
+			_update_dialog();
 			return;
 		}
 		memdelete(d);
 	}
 
-	FileAccess *f = FileAccess::create(FileAccess::ACCESS_RESOURCES);
-	create_new = !f->file_exists(p);
+	/* Does file already exist */
+
+	//	FileAccess *f = FileAccess::create(FileAccess::ACCESS_RESOURCES);
+	DirAccess *f = DirAccess::create(DirAccess::ACCESS_RESOURCES);
```

Comment:
Thx :) forgot to delete that one.

I use `f` here because I wanted to inform that this is file check not `dir` (like in original code).
I've even checked if `f` is not directory: 

`if (f->file_exists(p) && !(f->current_is_dir()))`

I use `DirAccess` only because it has methods I need to check if file is not directory.
`FileAccess` treated directories as files, hehe :B I didn't wanted to allocate and right after, delete memory, just so `ClassName` will match variable ^_^' and still wanted it to be readable that we are talking "files".

So, should I change `f` to `dir` or leave it as is? Or maybe add new tmp `FileAccess` variable?

Thx for review :3